docs/components/modeler/web-modeler/advanced-modeling/form-linking.md

```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
 This conversion will only be applied to the XML deployed to the cluster; the diagram in Web Modeler will not be changed.
 :::
 
+#### Known issues with linked forms
+
+##### Wrong form used for user task - [zeebe/#16311](https://github.com/camunda/zeebe/issues/16311)
+
+Some users have encountered an issue that was present in Camunda `8.4.0`, `8.4.1`, and `8.4.2` where linked forms did not correspond correctly to their tasks, leading to discrepancies in workflow execution. We have identified and rectified this issue in the Camunda `8.4.3` release, ensuring that tasks are now generated with the correct forms, as intended in your workflow design.
+
+###### How to fix the issue - Regenerate tasks with correct forms
+
+To correct any instances affected by this issue, we recommend the following steps after updating to Camunda `8.4.3`:
+
+1. Navigate to [Operate](/components/operate/operate-introduction.md) to access your workflow instances impacted by the linked form issue.
+2. Locate the instances where tasks were generated with the incorrect forms.
```

Comment:
Should the user then click on an instance? Click on the process instance key? How do I get to the screen with the wrench icon as mentioned in the next step?

Reply:
@christinaausley Updated.